src/Tasks/UnregisterAssembly.cs

```diff
@@ -4,6 +4,7 @@
 #if FEATURE_APPDOMAIN
 
 using System;
+using System.Diagnostics;
```

Comment:
Is the using necessary for this PR?

Reply:
It is: the calls to `[System.Diagnostics.]Debug.Assert` in the `#if`s I changed wouldn't compile without it.